jetstream/android-reengagment-notifications-release.toml

```diff
@@ -0,0 +1,17 @@
+[experiment]
+
+start_date = "2023-01-17"
+end_date = "2023-03-01"
+enrollment_period = 14
+
+[experiment.exposure_signal]
```

Comment:
This experiment is using the default exposure signal, implemented with `FxNimbus.features.reEngagementNotification.recordExposure()`

Do I need this section on the config, or is it automatic?